src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.ListView.cs

```diff
@@ -999,24 +999,42 @@ private void TasView_MouseWheel(object sender, MouseEventArgs e)
 			}
 		}
 
+		public void SetMarker(int frame = -1)
+		{
+			if (frame == -1) frame = Emulator.Frame;
+
+			TasMovieMarker existingMarker = CurrentTasMovie.Markers.FirstOrDefault(m => m.Frame == frame);

```

Comment:
```suggestion
			var existingMarker = CurrentTasMovie.Markers.FirstOrDefault(m => m.Frame == frame);
```
(I've been writing e.g. `TasMovieMarker/*?*/` where NRTs are disabled, but this doesn't need an explicit type IMO.)

Reply:
`var` is stupid imo. It is less readable in almost all situations and removes the ability to quickly navigate to definition or find other references since I can't put the cursor over the type name. In this particular case that's not such a big issue but I think it's better to always be explicit unless there's a special reason to not be.
I wouldn't mind using `/*?*/` though.